### ISSUE TITLE
Open Interfaces unexpected exception

### ISSUE THREAD

**Issue by DanielMurphy22** [mantidproject/mantid#28050]
**Original reporter:** Found by @Harrietbrown 

### Expected behavior
Should be able to open each interface
### Steps to reproduce the behavior
Open Plot or Workbench, open any of the following interfaces:
- [x] Diffraction>HFIR 4Circle
Following error trace:
```
Traceback (most recent call last):
  File "C:\MantidNightlyInstall\bin\workbench\app\mainwindow.py", line 417, in <lambda>
    action.triggered.connect(lambda checked_py, script=script: self.launch_custom_python_gui(script))
  File "C:\MantidNightlyInstall\bin\workbench\app\mainwindow.py", line 377, in launch_custom_python_gui
    self.interface_executor.execute(open(filename).read(), filename)
  File "C:\MantidNightlyInstall\bin\mantidqt\widgets\codeeditor\execution.py", line 155, in execute
    exec (code_obj, self.globals_ns, self.globals_ns)
  File "C:/MantidNightlyInstall/scripts/HFIR_4Circle_Reduction.py", line 12, in <module>
    from HFIR_4Circle_Reduction import reduce4circleGUI  # noqa
  File "C:\MantidNightlyInstall\scripts\HFIR_4Circle_Reduction\reduce4circleGUI.py", line 34, in <module>
    from HFIR_4Circle_Reduction import PreprocessWindow
  File "C:\MantidNightlyInstall\scripts\HFIR_4Circle_Reduction\PreprocessWindow.py", line 10, in <module>
    import reduce4circleControl
ModuleNotFoundError: No module named 'reduce4circleControl'
```
- [x] Direct>DGSPlanner

```
Traceback (most recent call last):
  File "C:\MantidNightlyInstall\bin\workbench\app\mainwindow.py", line 417, in <lambda>
    action.triggered.connect(lambda checked_py, script=script: self.launch_custom_python_gui(script))
  File "C:\MantidNightlyInstall\bin\workbench\app\mainwindow.py", line 377, in launch_custom_python_gui
    self.interface_executor.execute(open(filename).read(), filename)
  File "C:\MantidNightlyInstall\bin\mantidqt\widgets\codeeditor\execution.py", line 155, in execute
    exec (code_obj, self.globals_ns, self.globals_ns)
  File "C:/MantidNightlyInstall/scripts/DGSPlanner.py", line 14, in <module>
    planner = DGSPlannerGUI.DGSPlannerGUI()
  File "C:\MantidNightlyInstall\scripts\DGSPlanner\DGSPlannerGUI.py", line 55, in __init__
    self.instrumentWidget = InstrumentSetupWidget.InstrumentSetupWidget(self)
  File "C:\MantidNightlyInstall\scripts\DGSPlanner\InstrumentSetupWidget.py", line 220, in __init__
    self.updateFigure()
  File "C:\MantidNightlyInstall\scripts\DGSPlanner\InstrumentSetupWidget.py", line 274, in updateFigure
    self.gonfig.set_aspect('equal', adjustable='datalim')
  File "C:\MantidNightlyInstall\bin\lib\site-packages\matplotlib\axes\_base.py", line 1280, in set_aspect
    raise NotImplementedError(
NotImplementedError: It is not currently possible to manually set the aspect on 3D axes
```
- [x] Utility>QECoverage
```
Traceback (most recent call last):
  File "C:\MantidNightlyInstall\bin\workbench\app\mainwindow.py", line 417, in <lambda>
    action.triggered.connect(lambda checked_py, script=script: self.launch_custom_python_gui(script))
  File "C:\MantidNightlyInstall\bin\workbench\app\mainwindow.py", line 377, in launch_custom_python_gui
    self.interface_executor.execute(open(filename).read(), filename)
  File "C:\MantidNightlyInstall\bin\mantidqt\widgets\codeeditor\execution.py", line 153, in execute
    code_obj = compile(block.code_str, filename, mode=COMPILE_MODE,
  File "C:/MantidNightlyInstall/scripts/QECoverage.py", line 461
    def indirect_input_check(self, ana):
    ^
IndentationError: unexpected indent
```
(might this last one be fixed by #28048 ?)
QEcoverage DOES open in MantidPlot...
and it opens in a Workbench build of current master
### Platforms affected
Tested on 4.2.20200211.2150 Nightly Workbench and Plot

DGSPlanner issue does not affect Ubuntu.

**Comment by martyngigg** [comment]
If it helps the breakage of HFIR 4Circle is because of Python 3. Lines 10, 11 & 13 of  [PreprocessWindow.py](https://github.com/mantidproject/mantid/blob/master/scripts/HFIR_4Circle_Reduction/PreprocessWindow.py#L10) are missing the HFIR_4Circle_Reduction package prefix line line 12.

**Comment by martyngigg** [comment]
I suspect QECoverage to be fixed by #28048 

The `DGSPlanner` problem must be with matplotlib 3.1 I guess. Did this work on Windows before?

**Comment by martyngigg** [comment]
I think this is another set of system tests we want to add during maintenance: try to start every interface along with the workbench itself as mentioned in #28048.

**Comment by martyngigg** [comment]
I found matplotlib/matplotlib#/13474 which looks like the call never did the right thing so they've made it raise an error. A suggestion is to make the figure itself square if it is required but at minimum just removing the call should be enough.

**Comment by DanielMurphy22** [comment]
> I suspect QECoverage to be fixed by #28048
> 
> The `DGSPlanner` problem must be with matplotlib 3.1 I guess. Did this work on Windows before?

`DGSPlanner` opened fine in 4.2 and in Nightly on 131219 so presumeably its related to the matplotlib upgrade
